src/modules/mavlink/mavlink_log_handler.h

```diff
@@ -105,15 +65,42 @@ class MavlinkLogHandler
 	unsigned get_size();
 
 private:
+	enum class LOG_HANDLER_STATE {
+		INACTIVE,
```

Comment:
Can you clarify the difference between idle and inactive? Maybe use more descriptive names?

Reply:
@bkueng  I tried not to change the logic too much, not for now. so INACTIVE is similar to the case that the LogListHelper has not been created.
I added notes, and I know its a bit confusing, but without wider changes I don't want to change it much more.